gril/gril.c

```diff
@@ -573,16 +573,15 @@ static struct ril_msg *read_fixed_record(struct ril_s *p,
 	 */
 	g_assert(plen >= 8 && plen <= 4092);
 
-	/* If we don't have the whole fixed record in the ringbuffer
+	/*
+	 * If we don't have the whole fixed record in the ringbuffer
 	 * then return NULL & leave ringbuffer as is.
-	*/
+	 */
 
 	message_len = *len - 4;
 	if (message_len < plen)
 		return NULL;
 
-	/* FIXME: add check for message_len = 0? */
```

Comment:
Indeed, plen is guaranteed to be between 8-4092, so the comparison if (message_len < plen) will catch message_len == 0, and just return.